### ISSUE TITLE
Show status of files synced via a Cloud Provider

### ISSUE THREAD

**Issue by marbetschar** [elementary/files#1158]
## Prerequisites
- [x] I have searched open and closed issues for duplicates.

## Feature

Just came across the new Git status rendering of Files [on Twitter](https://twitter.com/mnavarrocarter/status/1216707760999620615) - that's really awesome!!

And there is the Cloud Providers plugin for Files which I'm also looking forward to (not tested yet):
https://github.com/elementary/files/pull/656

Now, are there any plans in adding a status Icon to CloudProvider files similar to the Git status to see if the file is either up to date, uploading or downloading? Similar to the way the Nextcloud application does it on macOS:

![external-content duckduckgo com](https://user-images.githubusercontent.com/392542/72442368-a3c94280-37ac-11ea-975b-578c33a97568.png)

**Comment by marbetschar** [comment]
In case anyone is interested to get Nextcloud integrations in Files right now, I made a short writeup of the steps involved over here: https://linux.marco.betschart.name/Desktop/Nextcloud-Client.html

**Comment by AlbertAlonso** [comment]
> In case anyone is interested to get Nextcloud integrations in Files right now, I made a short writeup of the steps involved over here: https://linux.marco.betschart.name/Desktop/Nextcloud-Client.html

@marbetschar I believe the correct link now is: [https://linux.marco.betschart.name/elementary-OS/Nextcloud-Client.html](https://linux.marco.betschart.name/elementary-OS/Nextcloud-Client.html)

**Comment by zeigerpuppy** [comment]
I don't think this works if you're not on element-OS.  I am using io.elementary.files on Debian (cinnamon desktop) with nextcloud-client installed.  the indicators for nextcloud don't show up in element files.
